backend/fms_core/models/sample.py

```diff
@@ -167,6 +168,9 @@ class Sample(TrackedModel):
     child_of = models.ManyToManyField("self", blank=True, through="SampleLineage",
                                       symmetrical=False, related_name="parent_of")
 
+    project = models.ForeignKey(Project, null=True, on_delete=models.PROTECT,
```

Comment:
To be removed from this PR; the association between sample and project has not be modelled yet, and it is not needed in this PR yet :)

Reply:
Perfect! Already removed :)